modules/evs/fsw/src/cfe_evs_task.h

```diff
@@ -89,6 +90,9 @@ typedef struct
     uint8  ActiveFlag;           /* Application event service active flag */
     uint8  EventTypesActiveFlag; /* Application event types active flag */
     uint16 EventCount;           /* Application event counter */
+    CFE_TIME_SysTime_t LastSquelchReturnedTime; /* Time of last squelch token return */
+    uint16             SquelchTokens;           /* Application event squelch token counter */
```

Comment:
Recommend using a signed type for the tokens.  This makes it simpler, if it goes negative, the events are squelched.  As long as its positive, its OK to send events.